packages/playground/worker/vite.config.ts

```diff
@@ -3,9 +3,10 @@ import { defineConfig } from 'vite'
 
 export default defineConfig({
   build: {
-    target: process.env.NODE_ENV === 'production' ? 'chrome60' : 'esnext'
+    // target: process.env.NODE_ENV === 'production' ? 'chrome60' : 'esnext'
```

Comment:
Is this change related to the esbuild issue @poyoho? Should we wait for it before merging this PR?

Reply:
I open a issues in vite #7179

and open in esbuild too. esbuild#2089

I think not. I think `chrome60` is not a important unit test.
The config may I copy from the issues repo in last time. 🙈

Reply:
Let's uncomment this line if it isn't needed then? Or remove the line directly?
We can merge it afterwards

Reply:
ok.